CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## Current release (in development)
 
+## 0.14.0 - 2021-2-15
+
+* Bind rubocop to => 0.9.0 to align with `pronto-rubocop`.
```

Comment:
You can add this to the `## Current release` section in #53, if you're so inclined. Otherwise, maintainers fill up the changelog when there is a release.